lib/matplotlib/figure.py

```diff
@@ -1277,7 +1351,7 @@ def subplots(self, nrows=1, ncols=1, sharex=False, sharey=False,
             labels of the bottom subplot are created. Similarly, when subplots
             have a shared y-axis along a row, only the y tick labels of the
             first column subplot are created. To later turn other subplots'
-            ticklabels on, use :meth:`~matplotlib.axes.Axes.tick_params`.
+            ticklabels on, use `~matplotlib.axes.Axes.tick_params`.
```

Comment:
Ummm, is this still true?  @ImportanceOfBeingErnest ? 

I actually think we should add a kwarg for this behaviour (and default it to True for back compatibility).  i.e. `hide_inner_ticklabels=True`; but thats out of scope here.

Reply:
Yes this is correct. You would use something like `axes[0,1].tick_params(labelbottom=True)` to turn the ticklabels on. Might be worth stating the parameter needed for this (`labelbottom`, `labelleft`) explicitely here?

(What does not work anymore is to use `plt.setp(axes[0,1].get_xticklabels(), visible=True)`, which was my long standing way of doing this.)